docs/src/joint/api/dia/attributes/containerSelector.html

```diff
@@ -0,0 +1,13 @@
+<p>The node denoted by the attribute value (a selector) will act as a proxy for the container (when the cell becomes the target of embedding). This will affect the following areas:
```

Comment:
```suggestion
<p>The value provided to this attribute is interpreted as a selector. The node matching this selector will act as a proxy for the container when the cell becomes the target of an embedding. This will affect the following areas:
```